packages/cli/src/builders.ts

```diff
@@ -167,6 +168,14 @@ const builders: SnapsCliBuilders = {
     demandOption: false,
     default: true,
   },
+
+  typescript: {
```

Comment:
How do you feel about replacing `--typescript` with `--javascript` and having it default to `false`?

Then it would be `mm-snap init` for TypeScript and `mm-snap init --javascript` for JavaScript.

If you have another suggestion, let me know. `mm-snap --typescript=false` seems confusing to me.

Reply:
As agreed in the [Slack discussion](https://consensys.slack.com/archives/C02GL38PLE6/p1655981110314169), this is changed to handle `--template typescript` or `--template javascript`. TypeScript is by default with only `mm-snap init`.